docs/product/guidelines/accessibility.html

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Accessibility
+description: A non-comprehensive guide to accessibility best practices when using Stacks.
+---
+
+<section class="stacks-section">
+    {% header "h2", "Overview" %}
+    <!-- TODO: Mention dry stuff - trying to meet WCAG 2.1 -->
+    <!-- TODO: State Stacks' goals/aspirations wrt a11y broadly. Think inclusive design principles https://inclusivedesignprinciples.org/ -->
+    <!-- TODO: Mention constant "work in progress" nature, "no such thing as 100% accessible" -->
+</section>
+
+<section class="stacks-section">
+    {% header "h2", "Accessibility testing" %}
+    <!-- TODO: Comment on how we test for accessibility. Mention mabl, dev tools, extensions, assistive technologies. -->
+</section>
+
+<section class="stacks-section">
+    {% header "h2", "Structural markup" %}
+    <!-- TODO: Comment on the benefits of using semantically precise structural markup -->
+</section>
+
+<section class="stacks-section">
+    {% header "h2", "Using WAI-ARIA" %}
+    <!-- TODO: Document best practices for using WAI-ARIA, mention visually hidden content (link to https://stackoverflow.design/product/base/visibility/#classes). Consider recommending an approach for aria-label vs visually hidden content. -->
+</section>
+
+<section class="stacks-section">
+    {% header "h2", "Color contrast" %}
+    <!-- TODO: Mention (and link to) high contrast and dark modes. Mention the benefits wrt accessibility. -->
+    <!-- TODO: Touch on limitations of WCAG 2.1 in this context. Maybe give a nod to WCAG 3.0 standards. -->
```

Comment:
See https://github.com/StackExchange/Stacks/pull/752#issuecomment-918652311 and https://github.com/StackExchange/Stacks/pull/752#issuecomment-923601203